README.adoc

```diff
@@ -127,6 +127,19 @@ NOTE: Debug output from `vagrant-service-manager` is prepended with
 the following string:
 `DEBUG command: [ service-manager: <command name / log message> ]`
 
+=== Configurations
+
+- HTTP Proxy Settings +
+User can able to start ADB in proxy environment by specifying following configurations
```

Comment:
I think you need to be a bit more verbose and explain the context. Something like:
"In a environment where HTTP traffic needs pass through a HTTP proxy, Docker and OpenShift need to be aware of the required proxy settings in order to function. _http_proxy_, _http_proxy_user_ and _http_proxy_password_ need to be specified. You can do so via: